common/utils/src/main/scala/org/apache/spark/internal/LogKey.scala

```diff
@@ -21,17 +21,57 @@ package org.apache.spark.internal
  * All structured logging keys should be defined here for standardization.
  */
 object LogKey extends Enumeration {
+  val EXECUTOR_ID = Value
```

Comment:
would it be better to sort alphabetically if we don't need to keep bianry compatible?

Reply:
+1, I just created https://issues.apache.org/jira/browse/SPARK-47705 for this.